code/modules/mining/equipment/kinetic_crusher/trophies_fauna.dm

```diff
@@ -0,0 +1,241 @@
+/*!
+ * Contains crusher trophies you can obtain from regular fauna
+ */
+
+//watcher
+/obj/item/crusher_trophy/watcher_wing
+	name = "watcher wing"
+	desc = "A wing ripped from a watcher. Suitable as a trophy for a kinetic crusher."
+	icon_state = "watcher_wing"
+	denied_type = /obj/item/crusher_trophy/watcher_wing
+	bonus_value = 5
+
+/obj/item/crusher_trophy/watcher_wing/effect_desc()
+	return "mark detonation to prevent certain creatures from using certain attacks for <b>[bonus_value*0.1]</b> second\s"
+
+/obj/item/crusher_trophy/watcher_wing/on_mark_detonation(mob/living/target, mob/living/user)
+	if(ishostile(target))
```

Comment:
i now realize this trophy has no effect on 98% mining mobs since they've all been refactored to basic mobs. but thats a bit unrelated to ur PR so i can fix it myself sometime.